### DISCUSSION TITLE
[Help] Updated to newest version, almost deleted all my work, and now I cannot save files in /bin

### DISCUSSION THREAD

**Discussion by aflores1030** [xournalpp/xournalpp#3329] [Q&A] [answered]
Sorry if I sound really out-of-it computer wise, most of the discussions on this site and sourceforge go way over my head, just a simple Xournal++ user and student.

Long story short, I had a Windows 10 Xournal++ version from August 2020 (I cannot recall the version sorry) and was wondering how to update it to the latest stable release. I figured I could just install the latest version, and it would automatically detect I was patching an existing installation. But to be on the safe side, I located the /bin folder with all my saved work, and copied it to an external folder just in case the update installation wiped it clean.

Well, it did do exactly that, so I was happy I had saved my work, and began the process of copying and pasting my work (autosaves and .xopp files, not any of the .dll files and stuff that came with the previous installation) into the new /bin folder. This required administrator access, which I provided.
Through this process I also used administrator access to rename the new installation's /bin folder but changed the name back after I was finished and before I started up the app.

Testing it, I found I could open and modify my old work in the new /bin no problem (yayy!), but I cannot save a new file at all in the /bin without the following error message:

**Save file error: Error opening file: "C:\ProgramFiles\Xournal++\bin\newfilename.xopp"**

Creating and saving a new file in the Documents folder on my computer works without a hitch, so I think it must be that the permissions on the /bin folder may have gotten screwed up when I was copying and pasting my old work into it. Any suggestions/questions for me?

Thank you so much.

**Comment by LittleHuba** [comment]
Can you start Xournal++ as admin? Right-click on the icon and select "Run as admin" (or something similar sounding). That should grant you the necessary rights.

Though I do highly recommend that you do not save all the documents in this directory. This directory is meant for the program files only. Any new version will remove any files in there and generally speaking, it is never a good idea to run a program as admin.

**Reply by aflores1030** [reply]
That actually made it work, though I see your point about this directory being only program files, and don't plan to run it as an administrator every time I open it up. Thanks

**Comment by rolandlo** [comment, marked as answer]
The solution is simple: Just don't put your own work into the `bin` folder. The `bin` folder is supposed to contain the binaries that can be run, not any user data. In general do not save anything into `C:\ProgramFiles`. Use your user profile folder (`C:\Users\<user>`) instead, where you need to replace `<user>` with your user name. There is a `Documents` folder within the user profile folder, which would be particularly fitting.

**Reply by aflores1030** [reply]
Yes, thank you, I'll just save files into a subfolder in Documents. 
Though it now seems odd to me that the older version had the bin under C:\Users\... and it seemed to be the default folder for storing new files, if it's not supposed to contain user data. Guess I'll move my old files into Documents as well, to make it easier to work with them. Thanks again.
